python/cudf/cudf/pylibcudf_tests/test_lists.py

```diff
@@ -198,6 +201,30 @@ def test_count_elements(test_data):
     assert_column_eq(expect, res)
 
 
+def test_apply_boolean_mask(list_column, bool_column):
+    arr = pa.array(list_column)
+    plc_column = plc.interop.from_arrow(arr)
+    plc_bool_column = plc.interop.from_arrow(bool_column)
+
+    res = plc.lists.apply_boolean_mask(plc_column, plc_bool_column)
+
+    expect = pa.array([[1], [2], [5], [6, 7]])
+
+    assert_column_eq(expect, res)
+
+
+def test_distinct():
```

Comment:
Can we parametrize this on `nulls_equal` and `nans_equal`?

Reply:
I parameterized `nulls_equal` but I remember getting unexpected results with `nans_equal` using `np.nan`. I don't think passing `nans_equal= True | False` changed the output using `np.nan` values in the list column. I'll look at it again...